Hazel/src/Hazel/Core.h

```diff
@@ -6,22 +6,29 @@
 	#else
 		#define HAZEL_API __declspec(dllimport)
 	#endif
+#elif HZ_PLATFORM_MACOS
+	#define HAZEL_API
 #else
-	#error Hazel only supports Windows!
+	#error Hazel does not support your platform!
 #endif
 
 #ifdef HZ_DEBUG
 	#define HZ_ENABLE_ASSERTS
 #endif
 
 #ifdef HZ_ENABLE_ASSERTS
-	#define HZ_ASSERT(x, ...) { if(!(x)) { HZ_ERROR("Assertion Failed: {0}", __VA_ARGS__); __debugbreak(); } }
-	#define HZ_CORE_ASSERT(x, ...) { if(!(x)) { HZ_CORE_ERROR("Assertion Failed: {0}", __VA_ARGS__); __debugbreak(); } }
+	#ifdef HZ_PLATFORM_WINDOWS
```

Comment:
Why separate the code when they are the same? `spdlog `should be cross platform already.

Reply:
Because there is no `__debugbreak();` on macOS.